css/moonstone-variables.less

```diff
@@ -251,6 +251,7 @@
 // ---------------------------------------
 @moon-input-border-width: 6px;
 @moon-input-height:60px;
+@moon-input-invalid-text-color: @moon-red;
```

Comment:
It seems like it might be better to either make this variable a bit more generic in name, i.e. `@moon-invalid-text-color`, or use `@moon-red` directly in the `moonstone/Input` style, otherwise it kind of defeats the purpose of creating this variable.